src/shared/sleep-config.c

```diff
@@ -102,6 +102,40 @@ int parse_sleep_config(SleepConfig **ret_sleep_config) {
         return 0;
 }
 
+int battery_is_low(void) {
+        int r;
+
+        r = read_battery_capacity_percentage();
+        if (r < 0)
+               return r;
+
+        if (r <= 5)
```

Comment:
shouldn't this be the other way around?

Reply:
this is a boolean, so 0 means "it is not low"

Reply:
ahh. okay. fixing it.